misc/translators/index.php

```diff
@@ -156,7 +156,7 @@
                                 <img src="https://a.ppy.sh/' . $translator['Id'] . '">
                                 <div class="translators__language-translator-texts">
                                     <p>' . $translator['Role'] . '</p>
-                                    <h1>' . $translator['Username'] . '</h1>
+                                    <h1>' . $translator['OsuUsername'] . '</h1>

```

Comment:
`$translator['OsuUsername'] ? $translator['OsuUsername'] : $translator['Username']` 
just in case theres some user that randomly translates stuff in crowding and/or doesn't have osu acc

Reply:
good point